recipes/xtensor-r/bld.bat

```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles" -D BUILD_TESTS=OFF -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
+nmake install
+if errorlevel 1 exit 1
```

Comment:
Is there a `ctest`?

Reply:
Yes, but xtensor, xtensor-r, xtensor-foobar all require C++14 to run the tests.

I will be running the tests as part of the recipes when the switch to gcc7 is done.